workflows/prognostic_c48_run/runtime/loop.py

```diff
@@ -133,6 +136,41 @@ def add_tendency(state: Any, tendency: State, dt: float) -> Tuple[State, State]:
     return updated, tendency_filled_frac  # type: ignore
 
 
+def _download_radiation_assets(
+    lookup_data_path: str = (
+        "gs://vcm-fv3gfs-serialized-regression-data/physics/lookupdata/lookup.tar.gz"  # noqa: E501
+    ),
+    forcing_data_path: str = "gs://vcm-fv3gfs-serialized-regression-data/physics/forcing/*",  # noqa: 501
```

Comment:
Are you concerend about this being inconsistent with the prog run? Would be nice to get it from the wrapper.

Reply:
Yeah, as mentioned above the port i/o would need to be rewritten to allow this, but it would be nice.